# High similarity among ChEC-seq datasets

### Chitvan Mittal1<sup>2</sup>, Matthew J. Rossi<sup>1</sup>, and B. Franklin Pugh<sup>1,2*</sup>

<sup>1</sup>Center for Eukaryotic Gene Regulation, Department of Biochemistry and Molecular Biology,
The Pennsylvania State University, University Park, PA 16802, USA

<sup>2</sup>Department of Molecular Biology and Genetics, Cornell University, Ithaca, New York, 14853,
USA

### Correspondence(*): fp265@cornell.edu

### [bioRxiv](https://www.biorxiv.org/content/10.1101/2021.02.04.429774v1)

## Abstract
ChEC-seq is a method used to identify protein-DNA interactions across a genome. It involves fusing micrococcal nuclease (MNase) to a protein of interest. In principle, specific genome-wide interactions of the fusion protein with chromatin result in local DNA cleavages that can be mapped by DNA sequencing. ChEC-seq has been used to draw conclusions about broad gene-specificities of certain protein-DNA interactions. In particular, the transcriptional regulators SAGA, TFIID, and Mediator are reported to generally occupy the promoter/UAS of genes transcribed by RNA polymerase II in yeast. Here we compare published yeast ChEC-seq data performed with a variety of protein fusions across essentially all genes, and find high similarities with negative controls. We conclude that ChEC-seq patterning for SAGA, TFIID, and Mediator differ little from background at most promoter regions, and thus cannot be used to draw conclusions about broad gene specificity of these factors.
